airbyte-integrations/connectors/source-s3/scripts/fetch_test_secrets.py

```diff
@@ -0,0 +1,46 @@
+"""Simple script to download secrets from GCS.
```

Comment:
Why not use the existing `get_config.py`?

Reply:
@natikgadzhi - I don't know what that is. 🫤